extensions/wikidata/tests/src/org/openrefine/wikidata/qa/scrutinizers/ScrutinizerTest.java

```diff
@@ -79,4 +92,33 @@ public void assertNoWarningRaised() {
     public Set<String> getWarningTypes() {
         return store.getWarnings().stream().map(w -> w.getType()).collect(Collectors.toSet());
     }
+
+    public void setFetcher(ConstraintFetcher fetcher) {
+        scrutinizer.setFetcher(fetcher);
+    }
+
+    public Stream<Statement> createStatementStream(PropertyIdValue propertyIdValue, EntityIdValue entityIdValue, PropertyIdValue property, Value propertyValue, PropertyIdValue item, Value itemValue) {
```

Comment:
The first parameter could be removed since it is always going to be P2302. The name of the method could be improved to suggest that those statements are constraints definitions.